src/router/routes/draft/finish.ts

```diff
@@ -56,6 +56,7 @@ const validateBody = (req: Request, res: Response, next: NextFunction) => {
   const { error } = FinishDraftBodySchema.validate(req.body);
   if (error) {
     res.status(400).json({ error: error.details[0].message });
+    return;
```

Comment:
Luckily I copied the Joi validation from this first and then kept getting headers already sent errors, because the next() function was still being called. I checked all the other new routers and only this one didn't return on validation error